src/hooks/useAddDoc.js

```diff
@@ -0,0 +1,21 @@
+import { useState } from 'react';
+import { db } from '../lib/firebase';
+import { collection, addDoc } from 'firebase/firestore';
+
+export default function useAddDoc() {
```

Comment:
We'll need to include the `itemName` variable used in line 12 as an argument to the function.